config/packages/lexik_jwt_authentication.yaml

```diff
@@ -0,0 +1,5 @@
+lexik_jwt_authentication:
+  secret_key:       '%kernel.project_dir%/config/jwt/private.pem' # required for token creation
+  public_key:       '%kernel.project_dir%/config/jwt/public.pem'  # required for token verification
+  pass_phrase:      'passphrase'
```

Comment:
Use the paths and passphrase from `.env`. Make sure to copy them to `.env.local`.